Portal/src/Datahub.Infrastructure/Queues/Messages/AppServiceConfigurationMessage.cs

```diff
@@ -0,0 +1,8 @@
+﻿using MediatR;
+
+namespace Datahub.Infrastructure.Queues.Messages
+{
+    public record AppServiceConfigurationMessage(
+        string ProjectAcronym
+        ) : IRequest;
```

Comment:
Is the message to update the workspace configuration off the database values or are we setting them via messages?

Either way it might make it clearer to add a verb in the object name around that, ie `WorkspaceAppServiceConfigurationRefreshMessage` or `WorkspaceAppServiceConfigurationUpdateMessage`

It's a bit verbose but it might clarify the usage and intent a bit